### ISSUE TITLE
Option to Attach Elastic Accelerator to ECS task

### ISSUE THREAD

**Issue by cloudinary-misha** [terraform-providers/terraform-provider-aws#11408]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Option to attach elastic inference accelerators to ECS tasks

According to this tutorial, AWS allows ECS to attach accelerators dynamically while running ECS cluster.
https://aws.amazon.com/blogs/machine-learning/running-amazon-elastic-inference-workloads-on-amazon-ecs/

### New or Affected Resource(s)

In order to make it work we need to add `inferenceAccelerators` block to 
aws_ecs_task_definition resource

```
"inferenceAccelerators": [
  {
    "deviceName": "device1",
    "deviceType": "eia1.medium"
  }
  ],
```

**Comment by ewbankkit** [comment]
Similar:
* https://github.com/terraform-providers/terraform-provider-aws/issues/10208
* https://github.com/terraform-providers/terraform-provider-aws/issues/8157

**Comment by bflad** [comment]
Support for this functionality in the `aws_ecs_task_definition` resource has been merged and will release with version 2.55.0 of the Terraform AWS Provider, later today. Thanks to @DrFaust92 for the implementation. 👍
